toolkits/landcover/impl/composites.js

```diff
@@ -67,16 +67,15 @@ function createTemporalComposites(
     // We count the 'date' band because it's been masked by all the others.
     var count = images.select('date').count().rename('observations');
 
-    return mosaic
-        .addBands(count)
+    return mosaic.addBands(count)
         .set('system:time_start', begin.millis())
         .set('system:id', begin.format('YYYYMMdd'))
         .set('system:index', begin.format('YYYYMMdd'));
   });
 
   // Filter out empty dates (no date band).
-  var filtered = ee.ImageCollection.fromImages(images)
-      .filter(ee.Filter.listContains('system:band_names', 'date'));
+  var filtered = ee.ImageCollection.fromImages(images).filter(
```

Comment:
When we're doing method chaining,prefer putting the '.' as the first character of the continuation line.

Reply:
Resubmitting this PR without clang applied.